src/core/server/saved_objects/service/lib/repository.ts

```diff
@@ -834,8 +850,10 @@ export class SavedObjectsRepository {
         (hit: SavedObjectsRawDoc): SavedObjectsFindResult => ({
           ...this._rawToSavedObject(hit),
           score: (hit as any)._score,
+          ...((hit as any).sort ? { sort: (hit as any).sort } : {}),
```

Comment:
I used `{}` so that it spreads out to nothing if `sort` isn't returned, because that's how fields are handled in the `rawToSavedObject` serializer:

https://github.com/elastic/kibana/blob/a87535624db850e8f9c4aafbc0796ab66d187d3b/src/core/server/saved_objects/serialization/serializer.ts#L82-L94

Although I could probably shorten this line to 
```ts
...((hit as any).sort && { sort: (hit as any).sort }),
```

Reply:
Oh I realize now you were probably asking about doing this instead:
```ts
...((hit as any).sort ? { sort: (hit as any).sort } : undefined),
```

Reply:
Yea, that was it 😄